coalib/bearlib/__init__.py

```diff
@@ -16,6 +16,10 @@ def deprecate_settings(**depr_args):
      ... def run(new):
```

Comment:
extra space after bracket in commit message.

also its For example: Having;
not For: Example having